applications/mlflow/charts/mlflow/templates/postgres-backup-deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mlflow.fullname" . }}-postgres-backup
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: db-backup-restore
+  template:
+    metadata:
+      labels:
+        app: db-backup-restore
+      annotations:
+        backup.velero.io/backup-volumes: backup
+        pre.hook.backup.velero.io/command: '["/scripts/db-backup.sh"]'
+        pre.hook.backup.velero.io/timeout: "10m"
```

Comment:
We'll need hook annotations for restore as well right?

Reply:
https://docs.replicated.com/vendor/snapshots-hooks#example

Reply:
I'm adopting the pattern I see in earlier KOTS version https://github.com/replicatedhq/kots/blob/710faf95e583827812a4ac41245bbf2e14b33af9/pkg/kotsadm/objects/kotsadm_objects.go#L873-L986 that we use `initContainer` for restore.

I just think with initContainer we has more control on dependency. E.g. wait for db ready --> restore --> app starts. With the restore annotation, I am not sure how it will get into the way of the underlying app.